include/cell/traits/b2b_gemm.hpp

```diff
@@ -9,7 +9,7 @@ using namespace cute;
 namespace tl = tile_layout;
 
 template <typename Element_, typename CtaTileShape,
-          typename WarpShape = tiledcuda::cell::TileShape<1, 1>,
+          typename WarpShape = tiledcuda::cell::TileShape<4, 1>,
```

Comment:
Add a comment for flash-attention-v2?

Reply:
but I think fused two dependent gemms is much general than flash attention.

Reply:
Ok, I got it.